docs/source/tools_generaltools.rst

```diff
@@ -487,6 +487,43 @@ To run the tool, use this Python command:
     import mmSolver.tools.reparent.tool as tool
     tool.unparent_to_world()
 
+.. _remove-solver-nodes-tool-ref:
+
+Remove Solver Nodes
+-------------------
+
+Remove Solver Nodes tool allows for the removal of 
+some or all nodes related to the matchmoveSolver 
+plugin, allowing for a clean scene to be prepped 
+to passed to other departmetns/vendors.
+
+.. figure:: images/tools_channel_box_sensitivity_ui.png
```

Comment:
To save your screenshot to the following directory, then `git add` the file to git:
`<project root>/docs/source/images/tools_remove_solver_nodes_ui.png`

Then change the line above to use it:
`.. figure:: images/tools_remove_solver_nodes_ui.png`

That's how you can add images to the documentation. I usually use PNGs, but JPEG, and GIF are also supported - whatever you need.